src/Bicep.Core/SemanticModel/NameBindingVisitor.cs

```diff
@@ -110,9 +131,13 @@ private Symbol ValidateFunctionFlags(Symbol symbol, TextSpan span)
             return symbol;
         }
 
-        private Symbol LookupSymbolByName(string name, TextSpan span)
+        private Symbol LookupSymbolByName(string name, TextSpan span, string? @namespace)
```

Comment:
Being explicit with @namespace to make sure callers pass or not a namespace value.